src/automation/run_3day_inventory.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash -l
+# This script is for running the automated inventory for 
+# the past 3 days to get updates for lagged data and plotting
+# the new data.
+
+SATINFO_DIR=/home/Jessica.Knezha/obs-inventory/build_gsinfo/satinfo
```

Comment:
what if we generalize it by using ${USER} or some other environmental variable so we dont have to have these hard-codded values in a github repo?

Reply:
Great idea. I have updated the scripts and tested them, it's fixed now.